database_schemas/cassandra/benchflow.cql

```diff
@@ -135,7 +139,7 @@ CREATE TABLE container_properties (
 );
 
 CREATE TABLE host_properties (
-  host_id uuid,
+  host_id text,
```

Comment:
We should always have our internal `uuid` id. Why in this case you suggest to directly use the one retrieved using Docker?